src/Parser/Rule/Package.idr

```diff
@@ -0,0 +1,69 @@
+module Parser.Rule.Package
+
+import public Parser.Lexer.Package
+import public Parser.Rule.Common
+
+import Data.List
+
+%default total
+
+public export
+PackageRule : Type -> Type
+PackageRule = Rule PackageToken
```

Comment:
Given that we already are in a module called `Package`, wouldn't it be better to name this just `Rule` and then use `Package.Rule` whenever it matters and just `Rule` whenever unambiguous?

`SourceRule` would then be `Source.Rule` or just `Rule` most of the time, too.

I think that module qualification is better than baking information into identifiers, both semantically and for convenience, too.

Reply:
Renamed {Package,Source}Token -> Token and {Package,Source}Rule -> Rule as per the suggestion, it does work quite nicely.